requirements/default.txt

```diff
@@ -2,4 +2,6 @@
 numpy>=1.7.1
 vtk>=8.1.2,!=9.0.0
 scipy>=1.2.0
-pillow>=5.4.1
+Pillow>=8.0.1
+aiohttp>=3.7.4
+aiortc
```

Comment:
should we make this module optional to the user?

Reply:
Maybe just webrtc protocol which depends on aiortc and ffmpeg. Although  the WebRTC streaming is the best way (or the only way) to share the streaming in a low-budget scenario like using ngrok.

The MJPEG streaming doesn't depend on anything special except aiohttp which is a very stable library. Thus I don't see why aiohttp and the MJPEG should be an optional module.